### DISCUSSION TITLE
Orthogonal Tensor Parameter (Moment) Maps

### DISCUSSION THREAD

**Discussion by bcmcpher** [dipy/dipy#3564] [Reconstruction]
## Description

I'm interested in working with a recently reported set of tensor derivatives that currently have an unclear path to creation with dipy.

Specifically, I want to work with the tensor parameter maps reported here:
https://www.sciencedirect.com/science/article/pii/S0197458020304310

Briefly, the tensor eigenvalues are summarized in 3 orthogonal metrics (mean diffusivity, MD; norm of anisotropy, NA; mode of anisotropy, MO) that describe the eigenvalue moments. Knowing MD already existed, I was hoping to contribute code to estimate the other 2.

However, after digging into what is currently available, I found that the components / operations needed exist already using a different notation from this paper:
https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.20741

I have a branch on my [fork](https://github.com/bcmcpher/dipy/tree/ten-orth-var) that adds an named function for creating NA in a way similar to existing metrics, as none of the current methods are equivalent to that map (MD / MO already exist). However, the notation from the reference does not match the existing documentation, so I'm unsure how to document it. 

I could also see this gap being handled with a new Documentation page detailing how to use the existing functions / notation to create the NA map instead of adding a new function for it.

I'm willing to contribute either (or both), I just need to know what would be preferred.

Adding @arokem, this is what I mentioned at the OHBM hackathon.

**Comment by skoudoro** [comment]
Hi @bcmcpher,

Thank you for request. 

I looked at [the commit in your branch](https://github.com/bcmcpher/dipy/commit/b73abf482974a231f61d7db4a7bdf60d34ef8a6c) and I am personally fine with both options.

The contribution should be in 2 distinct PRs. Concerning the documentation, I think you just need to update the [DTI tutorial](https://dipy.org/documentation/1.7.0/examples_built/07_reconstruction/reconst_dti/#sphx-glr-examples-built-07-reconstruction-reconst-dti-py) by adding and explaining those new parameters

**Comment by bcmcpher** [comment]
OK, I'll work up a PR with the some additional documentation as well. 

I thought of this as a new "Moments of the tensor" tutorial that would reuse a big part of the [DTI tutorial](https://dipy.org/documentation/1.7.0/examples_built/07_reconstruction/reconst_dti/#sphx-glr-examples-built-07-reconstruction-reconst-dti-py), but it could probably be integrated into the existing tutorial doc. 

The "challenge" that tripped me up was catching the different notation between the [current reference](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.20741) and the [new paper](https://www.sciencedirect.com/science/article/pii/S0197458020304310) - they're the same equations but have different notations. With that reconciled it shouldn't be too much to add.
